### ISSUE TITLE
Might be good to only squash merge the PR

### ISSUE THREAD

**Issue by xiki-tempula** [conda-forge/ambertools-feedstock#145]
### Comment:

The current repo is a bit bloated with commits that my laptop struggles with the git history. I think it might be better if we only squash merge PRs in the future.

**Comment by mikemhenry** [comment]
I agree, I don't know how to enforce this (since we don't get to control the settings on the repo) but I've done it for the last few PRs

**Comment by xiki-tempula** [comment]
Yes, I see it. Thanks for your effort.
